list.json

```diff
@@ -320,6 +320,35 @@
       "max": 2
     }
   },
+  {
+    "name": "x10-cm11-adapter",
+    "display_name": "X10 (CM11)",
+    "description": "X10 device support through the CM11 interface.",
+    "author": "Alan Thiessen",
+    "homepage": "https://github.com/AlanDThiessen/x10-cm11-adapter",
+    "version": "0.2.1",
+    "packages": {
+      "linux-arm64": {
```

Comment:
Did you intentionally only support `linux-arm64` and not `linux-arm`? The gateway image that we publish is a `linux-arm` image, so this would be unusable by default installations.

I'd be happy to add your repo to our add-on builder job if you need a `linux-arm` package built.

Reply:
I built the arm on a Raspberry Pi 3 Model B running the Mozilla IoT gateway image.  After doing some digging, I realized this is an `linux arm` architecture.  So I think I can just change it.  Is that correct?